xedocs/schemas/corrections/relative_light_yield.py

```diff
@@ -0,0 +1,20 @@
+'''
+# Correction: Relative light yield
+# What is it correction: Corrected areas (cS1)
+
+# Description: Correction done to the area of S1s in our detector. The relative light yield corresponds to the number of photons in the visible light spectrum released by an interaction per energy. This correction helps us convert from the number of photons observed in an S1 event to an energy.
+
+# Latest wiki reference: 
+
+'''
+
+
+import rframe
+
+from .base_corrections import TimeSampledCorrection
+
+class RelativeLightYield(TimeSampledCorrection):
+    
+    _ALIAS = "relative_light_yield"
+    value: float 
+    version: str = rframe.Index()
```

Comment:
same comment as before, the version field is already defined in the parent class so no need to redefine it unless you want to change its type/properties.
```suggestion
```